topics/transcriptomics/tutorials/goenrichment/tutorial.md

```diff
@@ -0,0 +1,359 @@
+---
+layout: tutorial_hands_on
+
+title: "GO Enrichment Analysis"
+zenodo_link: "https://zenodo.org/record/1255038#.Wx4qTBwh3CI"
+questions:
+  - "How can I functionally interpret a list of genes of interest that I obtained from my experiment?"
+objectives:
+  - "How to perform a GO Enrichment Analysis"
+  - "How to interpret and simplify the results"
+time_estimation: "1h"
+key_points:
+  - "The goenrichment tool can be used to perform GO Enrichment analysis"
+  - "One needs to be careful when chosing the background population"
+  - "There are several methods to simplify the output of the GO Enrichment analysis"
+contributors:
+  - igcbioinformatics
+  - mblue9
+---
+
+# Introduction
+{:.no_toc}
+
+When we have a large list of genes of interest (for example, a list of differentially expressed genes obtained from an RNA-Seq experiment), how do we extract biological meaning from that list?
+
+One way is to perform functional enrichment analysis. This method consists of the application of statistical tests to verify if genes of interest are more often associated to certain biological functions than what would be expected in a random set of genes. In this tutorial you will learn about enrichment analysis and how to perform it.
+
+**What is the Gene Ontology?** <br/>
+The [Gene Ontology](http://www.geneontology.org/) (GO) is a structured, controlled vocabulary and classification of gene function at the molecular and cellular level. It is divided in three separate sub-ontologies or GO types: biological process (e.g., signal transduction), molecular function (e.g., ATPase activity) and cellular component (e.g., ribosome). These sub-ontologies are structured as directed acyclic graphs (a hierarchy with multi-parenting) of GO terms.
+
+![](../../images/goenrichment_GOexample.png)
+>
+**Figure 1** QuickGO - http://www.ebi.ac.uk/QuickGO
+
+The GO Ontology, like other ontologies, are usually coded in the [OBO](http://www.geneontology.org/faq/what-obo-file-format) or the [OWL](http://www.geneontology.org/faq/what-owl-file) formats. It can be downloaded from the [Gene Ontology website](http://geneontology.org/page/download-ontology) or from the [OBO foundry](http://www.obofoundry.org/).
```

Comment:
Maybe we can also mention here that there are a bunch of tools to manipulate and work with OBO files?
This one for example: https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/onto_tk_obo2owl/onto_tk_obo2owl/1.45.0